charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
+
+## Definitions
+
+* A **Thematic Roadmap** identifies and expresses customer needs as a small collection of themes or strategic objectives which:
+    * Focuses the DCP community on a documented set of high-level customer priorities
+    * Excites users about the future of DCP
+
+* **Request for Comments (RFC)** is the transparent process that manages all significant enhancements to the DCP software technical and governance documentation, both normative and informative
+
+## In-scope
+
+*Governance* 
+
+* Develop standard processes and best practices for DCP community governance including (but not limited to):
+    * Program and technical oversight, decision making, and conflict resolution
+    * Charters for DCP projects
+    * Enhancements (RFC) to DCP software or governance 
+    * Planning
+    * Communication
+
+* Review and approve non-technical charters for DCP projects
+
+* Act as final arbiter for escalated conflicts from the DCP community
+
+*Communication*
+* Coordinate regular outreach and announcements to the DCP community and the HCA Science Leadership
+
+* Primary liaison between the HCA Science Leadership and the DCP community facilitated by the *Science Program Manager*
+
+*Planning*
+
+Strategic 
+
+* Identify and maintain a repository of user requirements, use cases, and journeys informed by *DCP UX* research
+
+* Facilitate a `<insert cadence>` planning process to solicit input from the HCA Science and DCP communities
+    * Publish a Thematic Roadmap RFC based on that input that also adddresses strategic technical objectives from the DCP Architecture team
+
+Tactical
+
+* Translate the customer priorities from the Thematic Roadmap RFC into actionable quarterly plans
```

Comment:
Something it would be good to discuss at the F2F meeting is how this ties to the charter roadmaps. Presumably they need to be aligned

Reply:
Plus we might now want tri-annual (given teritary has other meaning in the DCP context) plans if we have moved to three meetings a year

Reply:
Are you saying we'd do goals three times a year instead of four times a year? I hadn't heard we were thinking of that.

Reply:
I believe that Laura was correcting my _quarterly plans_ with _tri-annual plans_ - perhaps something like:

_Translate the customer priorities from the annual Thematic Roadmap RFC into actionable four month plans during each DCP meeting (which occurs three times a year)._

Reply:
Moving to three meetings a year was my idea to cut down on the travel overhead on the team. I was thinking that we would still do quarterly goals. The goals were usually put together by the PM and PI team anyway so we can hold smaller meetings for that. 

So I say meet en mass 3 times a year but still operate on quarterly goals.

Reply:
The advantage of having the goals tied to the meeting schedule though is it gives us a natural demarcation between sets of goals and a good opportunity to demonstrate the work done during the previous time period to the whole DCP. 

It would also be good to understand what the quarterly (or tertiary) goals are with respect to the charter milestones and the thematic roadmap. If we end up with to many different lists of things we are trying to achieve it will make our lives more difficult not easier.

Reply:
Personally, I like quarterly DCP meetings.  Coming from another project that demands monthly in person meetings quarterly seems like the right pace.

Reply:
It makes sense to me to update the thematic roadmap at the DCP meetings. I feel like we have a much better chance of ensuring alignment in person.

+1 to this proposal:
> Translate the customer priorities from the annual Thematic Roadmap RFC into actionable four month plans during each DCP meeting (which occurs three times a year).